### PR TITLE
248355: Creation of Platform variable for usage in flux

### PR DESCRIPTION
**Pull request by ajaysoni-cap**
# **What this PR does / why we need it**:
The PR has changes to create the Platform variables for consumption in the Flux kustomization files.
*Relates to ADO Work Item [AB#248355](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/248355) and builds on [386489](https://dev.azure.com/defragovuk/DEFRA-FFC/_build/results?buildId=386489&view=results)*

# **Special notes for your reviewer**
*Any specific actions or notes on review?*

# Testing
*Any relevant testing information and pipeline runs.*

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [x] Successful testing run(s) link provided
- [x] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:
gif

### CODE REVIEW

**Review by ppotturi — DISMISSED** [review]

**Review by DaRoszko — APPROVED** [review]

**Review by ppotturi — DISMISSED** [review]

File: infra/core/app-configuration/config-data/config-data.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      "value": "#{{ lower(environment) }}",
+      "label": "Platform"
+    },
+    {
+      "name": "ACR_NAME",
+      "value": "#{{ infraResourceNamePrefix }}#{{ nc_resource_containerregistry }}#{{ nc_instance_regionid }}01",
+      "label": "Platform"
+    },
+    {
+      "name": "NAMESPACE",
```
Comment (DaRoszko) [inline infra/core/app-configuration/config-data/config-data.json:13]: should prob be FLUX_NAMESPACE ?

File: infra/core/app-configuration/config-data/config-data.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      "value": "#{{ infraResourceNamePrefix }}#{{ nc_resource_servicebus }}#{{ nc_instance_regionid }}01",
+      "label": "Platform"
+    },
+    {
+      "name": "POSTGRES_SERVER_RG",
+      "value": "#{{ dbsResourceGroup }}",
+      "label": "Platform"
+    },
+    {
+      "name": "POSTGRES_SERVER",
```
Comment (DaRoszko) [inline infra/core/app-configuration/config-data/config-data.json:33]: Should we have POSTGRES_SERVER_{IDENTIFIER} ?? so if we add many servers etc.. servers for major programme etc?

**Review by DaRoszko — CHANGES_REQUESTED** [review]
would like a bit more descriptive var names

File: infra/core/app-configuration/config-data/config-data.json (hunk 1 above)
Comment (ajaysoni-cap) [reply infra/core/app-configuration/config-data/config-data.json:13]: Changed.

File: infra/core/app-configuration/config-data/config-data.json (hunk 2 above)
Comment (ajaysoni-cap) [reply infra/core/app-configuration/config-data/config-data.json]: Updated the variable to 1401 where 1401 is the identifier for the current server..

File: infra/core/app-configuration/config-data/config-data.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SHRD_POSTGRES_SERVER_RG",
+    "value": "#{{ dbsResourceGroup }}",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SHRD_POSTGRES_SERVER_#{{ nc_instance_regionid }}01",
```
Comment (DaRoszko) [inline infra/core/app-configuration/config-data/config-data.json:39]: why do we have region ID in here? should be a static key with a dynamic value no?

File: infra/core/app-configuration/config-data/config-data.json [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SERVICEBUS_RG",
+    "value": "#{{ servicesResourceGroup }}",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SERVICEBUS_NS",
```
Comment (DaRoszko) [inline infra/core/app-configuration/config-data/config-data.json:27]: would maybe add in SHRD_ at the start to follow convention ?

Comment (DaRoszko) [reply infra/core/app-configuration/config-data/config-data.json:27]: and make NS == NAMESPACE maybe ?

File: infra/core/app-configuration/config-data/config-data.json (hunk 3 above)
Comment (ppotturi) [reply infra/core/app-configuration/config-data/config-data.json]: Removed Region Id, Key Should be static

File: infra/core/app-configuration/config-data/config-data.json (hunk 4 above)
Comment (ppotturi) [reply infra/core/app-configuration/config-data/config-data.json]: Done

File: infra/core/app-configuration/config-data/config-data.json (hunk 2 above)
Comment (ppotturi) [reply infra/core/app-configuration/config-data/config-data.json]: Changed to SHARED_POSTGRES_SERVER_01

**Review by DaRoszko — DISMISSED** [review]

**Review by ppotturi — APPROVED** [review]

**Review by ppotturi — APPROVED** [review]

**Review by ppotturi — APPROVED** [review]

**Review by ppotturi — APPROVED** [review]

**Review by ppotturi — APPROVED** [review]

**Review by DaRoszko — APPROVED** [review]

**Pull request merged**